server/schedule/operator_controller.go

```diff
@@ -60,14 +59,15 @@ type HeartbeatStreams interface {
 // OperatorController is used to limit the speed of scheduling.
 type OperatorController struct {
 	sync.RWMutex
-	ctx             context.Context
-	cluster         opt.Cluster
-	operators       map[uint64]*operator.Operator
-	hbStreams       HeartbeatStreams
-	histories       *list.List
-	counts          map[operator.OpKind]uint64
-	opRecords       *OperatorRecords
-	storesLimit     map[uint64]*ratelimit.Bucket
+	ctx       context.Context
+	cluster   opt.Cluster
+	operators map[uint64]*operator.Operator
+	hbStreams HeartbeatStreams
+	histories *list.List
+	counts    map[operator.OpKind]uint64
+	opRecords *OperatorRecords
+	// TODO: Need to clean up the unused store ID.
```

Comment:
No need `TODO` here?

Reply:
TODO has done in next PR

Reply:
What I mean is the `TODO` here has already been done in the current master.

Reply:
OK